api/sheets/places_in.md

```diff
@@ -1,58 +1,38 @@
 ---
 layout: default
-title: Places within a Place
+title: Retrieving Child Nodes
```

Comment:
This function only works with place nodes. Please update the title here and below to reflect that.

Reply:
I changed it to "Retrieving Places Contained Within Another Place"